scripts/entrypoint.sh

```diff
@@ -1,10 +1,30 @@
 #!/bin/sh
 set -e
 
+# Create group and users
+addgroup --system --gid $PGID homarr
+adduser --system --uid $PUID --ingroup homarr external_user
+adduser --system --uid $INTERNAL_PUID --ingroup homarr internal_user
+
+# Change owner of existing directories
+chown -R $PUID:$PGID /secrets
+# Allow homarr group to write to secrets
+chmod -R g+w /secrets
+
 # Creating folders in volume
 mkdir -p /appdata/db
 mkdir -p /appdata/redis
 
-chown -R nextjs:nodejs /appdata
+chown -R $PUID:$PGID /appdata
+# Allow homarr group to write to appdata folder
+chmod -R g+w /appdata
+
+# Change owner to allow homarr to read docker socket when mounted
+DOCKER_SOCKET=/var/run/docker.sock
+if [ -r "$DOCKER_SOCKET" ]; then
```

Comment:
What happens if the socket is read-only? At least run a test

Reply:
![image](https://github.com/user-attachments/assets/385e3261-f28a-4536-afc7-6f7dc5bb19a9)
cant wait for it to fly in working, alread just changed the compose file